webapp/src/app/pages/components/knowledge-graph/cytoscape/cytoscape.component.ts

```
@@ -478,4 +527,34 @@
     }
     return all;
   }
+
+  toggleIrrelevantConcepts(cid: string, add: boolean = true) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)